### PR TITLE
basic path extracting

### PR DESCRIPTION
**Pull request by alexxingplus**

### CODE REVIEW

File: Tests/RegressionTests/PathExtractionSupport.swift [hunk 1]
```diff
@@ -0,0 +1,245 @@
...
+    case let (.lines(arr1), .lines(arr2)):
+      if arr1.count != arr2.count { return false }
+      let pairs = zip(arr1, arr2)
+      for (p1, p2) in pairs {
+        if !p1.isAlmostEqual(to: p2, tolerance: tolerance) {
+          return false
+        }
+      }
+      return true
+    default: return false
```
Comment (zienag) [inline Tests/RegressionTests/PathExtractionSupport.swift:227]: It's better to list all possible cases here for compiler to notice when new case will be added:
```
case (.moveTo, _), (. lineTo, _), (.curveTo, _) ...:
  return false
```

File: Tests/RegressionTests/SVGTests.swift [hunk 2]
```diff
@@ -201,6 +201,16 @@ class SVGCustomCheckTests: XCTestCase {
   }
 }
 
+class PathExtractionTests: XCTestCase {
+  func testLinesAndCurves() {
+    test(args: linesAndCurvesArgs)
+  }
+
+//  func testCirclesAndRects(){
+//    test(args: circlesAndRectsArgs)
```
Comment (zienag) [inline Tests/RegressionTests/SVGTests.swift:210]: uncomment or delete?

**Review by zienag — CHANGES_REQUESTED** [review]
Looks very promising! Just a couple improvements in comments

**Comment by alexxingplus** [comment]
I've just noticed failing `testNestedTransparentGroup` test, at the moment I don't quite understand why would it fail

**Review by zienag — APPROVED** [review]

**Pull request merged**